File: DSA/HR/mini-max_sum.java
import java.util.List;

class Result {
    public static void miniMaxSum(List<Integer> arr) {
        long min = arr.get(0), max = arr.get(0), total = 0;
        for (int num : arr) {
            if (num < min) {
                min = num;
            }
            if (num > max) {
                max = num;
            }
            total += num;
        }
        System.out.printf("%d %d\n", total - max, total - min);
    }

}

File: DSA/HR/mini-max_sum.cpp
#include <vector>
#include <iostream>

using namespace std;

void miniMaxSum(vector<int> arr)
{
    long long int min{arr[0]}, max{arr[0]}, total{arr[0]};

    for (int i{1}; i < arr.size(); ++i)
    {
        if (arr[i] < min)
            min = arr[i];
        if (arr[i] > max)
            max = arr[i];
        total += arr[i];
    }

    std::cout << total - max << " " << total - min << "\n";
}